### ISSUE TITLE
App freezes on splash screen

### ISSUE THREAD

**Issue by gerbil-snot** [hrydgard/ppsspp#12149]
iOS 11.2.6 iPhone 6 Plus

App is consistently freezing on the splash screen, never crashing and never opening (even left it there over night)

I can only ever get the app to open after a full reboot and only once. 

This issue persists with both dev-latest and dev-working

**Comment by unknownbrackets** [comment]
This issue is being closed as a duplicate of #9186.

This is an issue tracker - there's a [forum if you want to discuss things](http://forums.ppsspp.org/).  You wouldn't add another "bread" item to a grocery list every time you thought about it, right?  We do the same here - we already have "bread" (this issue) on our list.

By keeping the list clean, we can also make sure all the information a developer needs to fix the issue is at their finger tips.  When there are multiple issues, information gets lost and it becomes hard to fix things, so they are fixed more slowly.  Duplicates won't make fixes appear faster.

And remember - PPSSPP is open source.  Try to find *new and useful information*, and add it to the original issue.  If you can't wait any longer, muster up your courage and try fixing it yourself.  PPSSPP gets better the more people contribute.  You could be that amazing person who solves this problem for everyone.

-[Unknown]
